### DISCUSSION TITLE
Sentry <> FastAPI: Error Tracking and Performance Monitoring

### DISCUSSION THREAD

**Discussion by smeubank** [fastapi/fastapi#5220] [Show and tell]
### Announcement:

FastAPI is live, learn more from the following resources

[FastAPI Landing Page](https://sentry.io/for/fastapi/)
Blog: [FastAPI and Starlette Sentry Integrations Have Arrived](https://blog.sentry.io/2022/09/01/fastapi-and-starlette-sentry-integrations-have-arrived/)
[SDK Docs](https://docs.sentry.io/platforms/python/guides/fastapi/)

### What is available today?

- The integration is available today, latest versions of the Sentry Python SDK automatically detect the FastAPI framework and enable features to capture errors and monitor the performance of your FastAPI services
- Project onboarding experience:

<img width="1417" alt="image" src="https://user-images.githubusercontent.com/47563310/182374180-22986379-f726-4af9-9d79-54034cc8f8d0.png">

### What we would like to feedback on?

Any and all feedback, issues, PRs are welcome! We will respond in a timely manner on the [sentry-python SDK repo](https://github.com/getsentry/sentry-python/issues)

- What are your use cases, that are most important to you?
- Is there specific context for error tracking or performance monitoring that you would like for us to cover?

![fastapi-project](https://user-images.githubusercontent.com/47563310/182374491-11f814e2-cd41-4d32-8f9b-94719855a975.gif)

**Comment by nschuz** [comment]
amazing :)
